src/couchapps/WMStats/_attachments/js/DataStruct/T1/WMStats.RequestSummary.js

```diff
@@ -137,3 +137,24 @@ WMStats.Requests = function(data) {
     };
     return tier1Requests;
 };
+
+
+WMStats.Requests.getPropertyByTask = function(taskPath, property, requestInfo) {
+	
+    if (requestInfo.TaskChain) {
+        var numTasks = requestInfo.TaskChain;
+        var taskSplit = taskPath.split("/");
+        var task = taskSplit[taskSplit.length -1];
+        for (var i=1; i <= numTasks; i++) {
+            if (requestInfo["Task" + i].TaskName == task) {
+                if (requestInfo["Task" + i][property] !== undefined) {
```

Comment:
I'm not sure if we should also check for null value here. It looks like the answer is no, since most of the properties we'd look for don't have a default value inside the tasks:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WMSpec/StdSpecs/TaskChain.py#L554